### ISSUE TITLE
CIDR range of IP addresses in text files

### ISSUE THREAD

**Issue by Amirziplin** [angryip/ipscan#404]
plase add  CIDR  range of IP addresses in  scan from Text file

**Comment by GgBoom-993** [comment]
Add 1. I must admit it's powerful，but It is odd that a large-scale IP scanning tool does not support the goal of CIDR

**Comment by bs20707** [comment]
Agreed having a one IP per line for this to text file mode to work seems a bit 1980s-ish. CDIR or some other formatted means would go a long way. Would be nice if default ports could be specified as well, but first things first.
